### PR TITLE
Testing updates 1

### PR DESCRIPTION
**Pull request by jasonmimick**
## Description

Please include a summary of the fix/feature/change, including any relevant motivation and context.

Link to any related issue(s): 

## Type of change:

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Required Checklist:

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation (if appropriate)
- [ ] I have run `make fmt` and formatted my code

### CODE REVIEW

**Comment by jasonmimick** [comment]
This is ready for CR. Fixes an issue in cluster which will cause nil pointer.

**Comment by jasonmimick** [comment]
Note - there might be an issue with this, somehow there is a change from "ReplicationSpecs" to "replicationSpecs".
I'm checking that and will update - hold on CR until update

**Comment by jasonmimick** [comment]
Ok - PR updated, validated and tested by running `test-type` in us-east-1 and published all resources.
This is ready for review.

File: cfn-resources/cfn-publishing-helper.config [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export CFN_PUBLISH_TAG=$(git rev-parse --short HEAD)
+export CFN_TEST_LOG_BUCKET="mdb-cfn-publish-${CFN_PUBLISH_TAG}"
+export CFN_PUBLISH_MAJOR_VERSION=1
+export CFN_PUBLISH_MINOR_VERSION=0
+export AWS_DEFAULT_PROFILE=711489243244_AdministratorAccess
```
Comment (coderGo93) [inline cfn-resources/cfn-publishing-helper.config:5]: is this always the same or it can be changed?

**Review by coderGo93 — APPROVED** [review]
one small question but if the tests works then LGTM

**Pull request merged**